examples/bobandlynn.jsonld

```diff
@@ -0,0 +1,144 @@
+{
+    "@context": {
```

Comment:
Don't know @ahdinosaur , I just embedded it as easier for me.  Neither of them works because there is no vf.  How about if we get the vf working, and then everyone reference that?

I will take out the foaf references though and use all vf.